api/nakadi-event-bus-api.yaml

```diff
@@ -574,6 +574,25 @@ paths:
             The flow id of the request, which is written into the logs and passed to called services. Helpful
             for operational troubleshooting and log analysis.
           type: string
+        - name: X-nakadi-cursors
```

Comment:
It is not mandatory (http://zalando.github.io/restful-api-guidelines/naming/Naming.html), but I think it is pretty common to use hyphenated pascal case for headers.